pilot/pkg/bootstrap/server.go

```diff
@@ -438,6 +440,29 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		}()
 	}
 
+	if s.MultiplexGRPC {
```

Comment:
I think we should use 'GRPCPort==""' - if the dedicated gprc port is disabled, we multiplex.

Reply:
we do set MultiplexGRPC to be true iff `GRPCPort==""`. So I think its the same in practice, that may be a bit simpler though?